### PR TITLE
Add head_vcf.py script

### PR DESCRIPTION
**Pull request by xbrianh**

### CODE REVIEW

File: scripts/head_vcf.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    raw = gscio.Reader(blob)
+else:
+    raw = open(args.cloudpath, "rb")
+
+with closing(raw):
+    with bgzip.BGZipReader(raw) as bgreader:
+        with io.BufferedReader(bgreader) as reader:
+            for line in reader:
+                if line.startswith(b"#"):
+                    print(line.decode("ascii").strip())
```
Comment (DailyDreaming) [inline scripts/head_vcf.py:27]: Are these guaranteed to be `ascii`?

File: scripts/head_vcf.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    blob = Client().bucket(bucket_name).get_blob(key)
+    raw = gscio.Reader(blob)
+else:
+    raw = open(args.cloudpath, "rb")
+
+with closing(raw):
+    with bgzip.BGZipReader(raw) as bgreader:
+        with io.BufferedReader(bgreader) as reader:
+            for line in reader:
+                if line.startswith(b"#"):
```
Comment (DailyDreaming) [inline scripts/head_vcf.py:26]: I could be misremembering, but I feel like I've seen something like:

```
# header header
# more header

# casual line skipped
actual vcf contents
```

Would it be good to also skip `line.strip() == ''` and only break if it hits content?

File: scripts/head_vcf.py (hunk 1 above)
Comment (xbrianh) [reply scripts/head_vcf.py:27]: Actually, this should be utf-8 according to [the spec](http://samtools.github.io/hts-specs/VCFv4.3.pdf)

Thanks!

File: scripts/head_vcf.py (hunk 2 above)
Comment (xbrianh) [reply scripts/head_vcf.py:26]: will do

**Comment by xbrianh** [comment]
@DailyDreaming I'm going to merge this based on manual testing, but see [this issue](https://github.com/DataBiosphere/terra-notebook-utils/issues/19)

**Comment by DailyDreaming** [comment]
@xbrianh Sounds good!

**Pull request merged**